packages/strapi-plugin-i18n/services/content-types.js

```diff
@@ -73,7 +74,9 @@ const isLocalizedAttribute = (model, attributeName) => {
  */
 const getNonLocalizedAttributes = model => {
   return Object.keys(model.attributes)
-    .filter(attributeName => !['locale', 'localizations'].includes(attributeName))
+    .filter(
+      attributeName => !['locale', 'localizations', PUBLISHED_AT_ATTRIBUTE].includes(attributeName)
```

Comment:
I think it makes sense to remove all the nonVisible fields . what do you think ?